.ruby-version

```diff
@@ -1 +1 @@
-2.1.3
+2.1.2
```

Comment:
@canes6288 mind reverting this change? Though we're now stable at 2.2.0; perhaps we can update our ruby and ember builds in a future commit, if we're being bold.